packages/user-banning-nodejs/src/config.ts

```diff
@@ -0,0 +1,6 @@
+export const PLUGIN_ID = 'supertokens-plugin-user-banning';
+export const PLUGIN_SDK_VERSION = ['23.0.0', '23.0.1'];
```

Comment:
It should probably also include `>=23.0.1` as well (both here and the frontend) for forward compatibility.